### PR TITLE
[Meta] Gradle tasks for version bumping

### PR DESCRIPTION
**Pull request by Elyniss**

### CODE REVIEW

File: buildSrc/src/main/kotlin/version-bump.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,62 @@
+// [x] Wszystko ładnie ma działać za odpaleniem jednego skryptu
+// [ ] Udokumentować co jeszcze trzeba zrobić żeby łądnie działało w CI
+// [x] Dodać testy -> kod wywalić z tego pliku do osobnego pliku i testować
+// [x] zmienić readme aby miało - zamist kropki w czacie
+// [ ] ogarnać jak zrobić release z GHA
```
Comment (TalhaAli00) [inline buildSrc/src/main/kotlin/version-bump.gradle.kts:5]: Can we just keep the points that are needed?

File: product/walletconnectmodal/ReadMe.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ allprojects {
 app/build.gradle.kts
 
 ```gradle
-implementation(platform("com.walletconnect:android-bom:$BOM_VERSION"))
+implementation(platform("com.walletconnect:android-bom:$example.BOM_VERSION"))
```
Comment (TalhaAli00) [inline product/walletconnectmodal/ReadMe.md:40]: what is example?

File: product/web3modal/ReadMe.md [hunk 3]
```diff
@@ -33,7 +33,7 @@ allprojects {
 app/build.gradle.kts
 
 ```gradle
-implementation(platform("com.walletconnect:android-bom:$BOM_VERSION"))
+implementation(platform("com.walletconnect:android-bom:$example.BOM_VERSION"))
```
Comment (TalhaAli00) [inline product/web3modal/ReadMe.md:36]: same

File: buildSrc/src/main/kotlin/version-bump.gradle.kts (hunk 1 above)
Comment (Elyniss) [reply buildSrc/src/main/kotlin/version-bump.gradle.kts:5]: Lol, forgot to remove

File: sample/common/build.gradle.kts [hunk 4]
```diff
@@ -15,7 +15,7 @@ android {
         consumerProguardFiles("consumer-rules.pro")
         buildConfigField("String", "PROJECT_ID", "\"${System.getenv("WC_CLOUD_PROJECT_ID") ?: ""}\"")
         buildConfigField("String", "MIX_PANEL", "\"${System.getenv("MIX_PANEL") ?: ""}\"")
-        buildConfigField("String", "BOM_VERSION", "\"$BOM_VERSION\"")
+        buildConfigField("String", "example.BOM_VERSION", "\"$BOM_VERSION\"")
```
Comment (TalhaAli00) [inline sample/common/build.gradle.kts:18]: same

File: sample/dapp/build.gradle.kts [hunk 5]
```diff
@@ -23,7 +23,7 @@ android {
             useSupportLibrary = true
         }
         buildConfigField("String", "PROJECT_ID", "\"${System.getenv("WC_CLOUD_PROJECT_ID") ?: ""}\"")
-        buildConfigField("String", "BOM_VERSION", "\"${BOM_VERSION ?: ""}\"")
+        buildConfigField("String", "example.BOM_VERSION", "\"${BOM_VERSION ?: ""}\"")
```
Comment (TalhaAli00) [inline sample/dapp/build.gradle.kts:26]: same

File: product/walletconnectmodal/ReadMe.md (hunk 2 above)
Comment (Elyniss) [reply product/walletconnectmodal/ReadMe.md:40]: nice catch. I need to remove it

File: sample/modal/build.gradle.kts [hunk 6]
```diff
@@ -22,7 +22,7 @@ android {
             useSupportLibrary = true
         }
         buildConfigField("String", "PROJECT_ID", "\"${System.getenv("WC_CLOUD_PROJECT_ID") ?: ""}\"")
-        buildConfigField("String", "BOM_VERSION", "\"${BOM_VERSION}\"")
+        buildConfigField("String", "example.BOM_VERSION", "\"${BOM_VERSION}\"")
```
Comment (TalhaAli00) [inline sample/modal/build.gradle.kts:25]: same

File: sample/wallet/build.gradle.kts [hunk 7]
```diff
@@ -22,7 +22,7 @@ android {
             useSupportLibrary = true
         }
         buildConfigField("String", "PROJECT_ID","\"${System.getenv("WC_CLOUD_PROJECT_ID") ?: ""}\"")
-        buildConfigField("String", "BOM_VERSION", "\"${BOM_VERSION}\"")
+        buildConfigField("String", "example.BOM_VERSION", "\"${BOM_VERSION}\"")
```
Comment (TalhaAli00) [inline sample/wallet/build.gradle.kts:25]: same

**Review by TalhaAli00 — APPROVED** [review]

**Review by jakubuid — APPROVED** [review]
Great work!
Let's document how and when to use version bump commands, what are the differences between, RELEASE, FIX, MANUAL types. If we add a new module, how to include it in this process? Notion page with all of that would be great to have.

**Comment by Elyniss** [comment]
Added to our release notion doc

**Pull request merged**